app/src/renderer/components/staking/LiDelegate.vue

```diff
@@ -1,16 +1,25 @@
 <template lang='pug'>
 .li-delegate(:class='styles'): .li-delegate__values
   .li-delegate__value.name
-    router-link(:to="{ name: 'validator', params: { validator: delegate.id }}") {{ delegate.description.moniker }}
-  .li-delegate__value.percent_of_vote
-    span {{ delegate.percent_of_vote }}
-  .li-delegate__value.number_of_votes.num.bar
-    span {{ num.pretty(delegate.voting_power) }}
-    .bar(:style='vpStyles')
+    router-link(:to="{ name: 'validator', params: { validator: delegate.id }}")
+      img.avatar(v-if="delegate.keybase" :src="delegate.keybase.avatarUrl" width="48" height="48")
+      img.avatar(v-else src="~assets/images/validator-icon.svg" width="48" height="48")
+      .vert
+        .top {{ delegate.description.moniker }}
+        .bottom {{ shortAddress(delegate.id)}}
   .li-delegate__value.your-votes
     span {{ yourVotes }}
-  .li-delegate__value.status
-    span {{ delegateType }}
+  .li-delegate__value.your-rewards
+    span {{ yourRewards }}
+  .li-delegate__break: span
+  .li-delegate__value.percent_of_vote
+    span {{ delegate.percent_of_vote }}
+  .li-delegate__value.uptime
+    span.NOT_green {{ uptime }}
```

Comment:
what is this `NOT` about?

Reply:
I didn't quite get that.  Would you please explain it again in German?

Reply:
I guess those are placeholders. I replace them with comments.